src/main/resources/rhizome.yaml

```diff
@@ -14,10 +14,8 @@ hazelcast:
         - "localhost"
     port: 5710
 
-postgres:
-    citus: false
-    hikari:
-        jdbcUrl: "jdbc:postgresql://localhost:5432/openlattice"
-        username: "oltest"
-        password: "test"
-        maximumPoolSize: 3
+hikari:
+    jdbcUrl: "jdbc:postgresql://localhost:5432/openlattice"
+    username: "oltest"
+    password: "test"
+    maximumPoolSize: 3
```

Comment:
This will break things. `RhizomeConfiguration` expects a `postgres` property:

https://github.com/geekbeast/rhizome/blob/develop/src/main/java/com/kryptnostic/rhizome/configuration/RhizomeConfiguration.java
https://github.com/geekbeast/rhizome/blob/develop/src/main/kotlin/com/geekbeast/configuration/postgres/PostgresConfiguration.kt